src/calendar-list/item.js

```diff
@@ -47,13 +71,13 @@ class CalendarListItem extends Component {
           disabledByDefault={this.props.disabledByDefault}
           showWeekNumbers={this.props.showWeekNumbers}
           renderArrow={this.props.renderArrow}
-          onPressArrowLeft={this.props.onPressArrowLeft}
-          onPressArrowRight={this.props.onPressArrowRight}
+          onPressArrowLeft={this.onPressArrowLeft}
```

Comment:
I would limit this fix to horizontal only, so we won't break the current behaviour of CalendatList in vertical for those who choose to add the navigation arrows for some reason. Consider something like:
`onPressArrowLeft={this.props.horizontal ? this.onPressArrowLeft : this.props.onPressArrowLeft}
  onPressArrowRight={this.props.horizontal ? this.onPressArrowRight : this.props.onPressArrowRight}`

Reply:
@Inbal-Tish Okay, I've made these changes.